src/Storage/Stack/Storage.sln

```diff
@@ -49,4 +69,9 @@ Global
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
+	GlobalSection(NestedProjects) = preSolution
+		{D6F470A6-7395-4B8B-9D29-44DF0EC8F624} = {4A88F402-7A6A-4E36-B339-08C81E5F7876}
+		{3B48A77B-5956-4A62-9081-92BA04B02B27} = {4A88F402-7A6A-4E36-B339-08C81E5F7876}
+		{C1BDA476-A5CC-4394-914D-48B0EC31A710} = {4A88F402-7A6A-4E36-B339-08C81E5F7876}
+	EndGlobalSection
```

Comment:
ALso, please update the encoding of AzureRM.psm1 below - it should be using UTF-8 encoding

Reply:
Updated, unintended change, thanks for catching it